cmd/server/main.go

```
@@ -1,3 +1,33 @@
+
+	router := chi.NewRouter()
+	store := store.NewMetricStore()
+	handlers := handlers.NewHandler(store)
+
+	router.Use(middleware.WithLogger(logger))
+
+	router.Post("/update/{type}/{name}/{value}", handlers.HandlePOSTMetric)
+	router.Get("/", handlers.HandleShowAllMetrics)
+	router.Get("/value/{type}/{name}", handlers.HandleShowMetricValue)
+
+	logger.InfoContext(nil, "Server started at ", slog.String("address", srvConfig.Address))
```

golangci-lint (1.62.2)

SA1012: do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use

(staticcheck)